docs/sphinx/user_guide/feature/view.rst

```diff
@@ -82,6 +82,44 @@ accesses array entries with unit stride. The loop::
 
 access array entries with stride N :subscript:`n` * N :subscript:`(n-1)` * ... * N :subscript:`(j+1)`.
 
+MultiView
+^^^^^^^^^^^^^^^^
+
+A ``RAJA::MultiView`` object wraps an array-of-pointers,
+or a pointer-to-pointers, rather than a ``RAJA::View`` which wraps a single

```

Comment:
This is awkward. Suggested change: "rather than" -> "whereas" (or "while")

Reply:
For little things like this, we have recently started requesting folks on the OpenMP repo use the "suggested change" feature, might save some effort over here too, like this:

```suggestion
or a pointer-to-pointers, whereas a ``RAJA::View`` which wraps a single

```

Reply:
Thanks @trws I was not aware of that feature. I will use it in the future.

Reply:
It's only newly available everywhere. Bronis and I just gave a little tutorial on it for the OpenMP committee so we could start using it for the small stuff where it's easier to just make the change than explain it.  Probably should have thought of it during the meeting this morning but didn't occur to me.  It's especially nice since they added the "Add suggestion to batch" button, so you can aggregate all the little suggestions into a batched commit.

Reply:
👍